kore.maude

```diff
@@ -4,12 +4,13 @@ mod KORE is
 --- For example, we cannot write `_+_` in Maude; so we'll write #Label("_+_").
 --- For maximum clarity, we will also include a separate sort for each type of token.
   including STRING .
-  sorts Label Token Variable Application Item K KList .  --- we call these "K Label", etc.
+  sorts Label Token Variable Application Item K KList Sort .  --- we call these "K Label", etc.
   subsorts Token Variable Application < Item < K < KList .
   op #Label : String -> Label .
-  op #Token : String String -> Token .
-  op #Variable : String String -> Variable .
-  op _`(_`) : Label KList -> Application .
+  op #Token : Sort String -> Token . --- Cosmin: if we have sorts, it may be good to have them explicitly "String String" is confusing
+  op #Sort : String -> Sort .
+  op #Variable : Sort String -> Variable .
```

Comment:
Good idea to move the Sort declaration here, and then to use the actual sort construct in variables and tokens.